src/ansys/fluent/core/utils/singleton_meta.py

```diff
@@ -0,0 +1,20 @@
+"""Provides a module for Metaclasses."""
```

Comment:
Shall we move this to `src/ansys/fluent/core/meta.py` ?

Reply:
Don't think so. meta.py has some very definite purpose and it should not contain generic methods.

Reply:
@hpohekar asks a very useful question. I believe that we should only either agree or propose an alternative. We can see by searching the code that all these classes are only used in the context of **external** tools for postprocessing, visualization and user interfaces. In the codebase currently, these classes are located amongst general-purpose PyFluent code in modules headed by docstrings like "this is generic code", which doesn't help to clarfiy the intent nor the current usage. It's also unlikely that any of this code will be reused directly in PyFluent itself. We already have the separate `post_objects` namespaces which is serving a similar purpose. We *could* rename that folder and put all the aforementioned code in there.

Reply:
@seanpearsonuk Just now I discussed this point with @ajain-work. We can remove `singleton_meta.py` because we are not using it anywhere in PyFluent.

We can move  `src/ansys/fluent/core/meta.py` to `post_objects` because it is visualization specific only.

Reply:
@hpohekar class in `singleton_meta.py` are used in pyfluent-visualization.

Reply:
@seanpearsonuk Right. I discussed this point with @ajain-work again. So he suggested that we can keep both `singleton_meta.py` and `meta.py` under `post_objects`. So we will also move separated modules under `post_objects` namespace. Do we want to rename `post-objects` ?

Reply:
Renaming `post-objects` seems like a good option, as mentioned in an earlier post.